### ISSUE TITLE
Image URL mismatch between hero and OpenGraph

### ISSUE THREAD

**Issue by ge0rg** [danielkvist/hugo-terrassa-theme#32]
Hugo will [generate Open Graph meta-data by default](https://gohugo.io/templates/internal/#open-graph), using the `images` property, among others. It will assume that the image is located in the site's root directory. If I have the following header in `_index.md`:

```md
---
images: ["test.jpg"]
---
```

Hugo will generate the following HTML:

```html
<meta name="twitter:image" content="https://mysite.local/test.jpg"/>
<meta property="og:image" content="https://mysite.local/test.jpg" />
```

Terrassa, on the other hand, will generate this:

```
<main class="hero" 
    style="background-image: url('https://mysite.local/images/test.jpg');" >
```

There is no way to consolidate those two things, except for putting the image into two different places. Furthermore, using the hero image as the OG preview is sub-optimal.

The least horrible workaround right now is to use the same file name for different purposes:
- /test.jpg is the OG preview
- /images/test.jpg is the hero graphic.

**Comment by feministclickback** [comment]
Hey, yes, the template for the hero section adds the /images/ to the file path. I agree that the hero image is a bad choice for the og image. 

One way to solve this imo is to 

- add another frontmatter variable heroImage
- adjust the template to use this variable instead of "images"

Does that solve the problem in your view?

**Comment by ge0rg** [comment]
Yeah, using a different variable for the hero image would solve the problem for me. It would probably also make sense to remove the /images/ prefix while you are at it :)
